### PR TITLE
Add body and filter fields to GlobalWebhook schema

### PR DESCRIPTION
**Pull request by ninjasimon**
## Summary
Adds `body` and `filter` fields to GlobalWebhook schema.

## Checklist

- [X] Writing style
- [X] API design standards

### CODE REVIEW

File: openapi/components/schemas/GlobalWebhook.yaml [hunk 1]
```diff
@@ -38,6 +38,21 @@ properties:
   credentialHash:
     type: string
     description: Hash from credentials to use for authentication by the given URL.
+  body:
+    description: |-
+      Webhook request body.
+      If null, the default payload for the given event is used.
```
Comment (adamaltman) [inline openapi/components/schemas/GlobalWebhook.yaml:44]: This doesn't explain or link to how someone can use placeholders.

File: openapi/components/schemas/GlobalWebhook.yaml [hunk 2]
```diff
@@ -38,6 +38,21 @@ properties:
...
+    type: string
+    nullable: true
+  filter:
+    description: |-
+      Filter that determines whether to send the webhook.
+      This field requires a special format. Use `,` for multiple allowed values.
```
Comment (adamaltman) [inline openapi/components/schemas/GlobalWebhook.yaml:50]:
```suggestion
      This field requires a special format.
      Use `,` for multiple allowed values.
```

File: openapi/components/schemas/GlobalWebhook.yaml (hunk 1 above)
Comment (ninjasimon) [reply openapi/components/schemas/GlobalWebhook.yaml:44]: I added additional information about placeholders.

File: openapi/components/schemas/GlobalWebhook.yaml [hunk 3]
```diff
@@ -38,6 +38,29 @@ properties:
...
     type: string
     description: Hash from credentials to use for authentication by the given URL.
+  body:
+    description: |-
+      Webhook request body.
+      If null, the default payload for the given event is used.
+      Template placeholders are permitted.
+
+      Invalid placeholders render as an empty strings. For example, `Hello
+      {{invalid.placeholder}}!` is rendered as `Hello !`.
```
Comment (adamaltman) [inline openapi/components/schemas/GlobalWebhook.yaml:48]:
```suggestion
      Invalid placeholders render as an empty strings.
      For example, `Hello {{invalid.placeholder}}!` is rendered as `Hello !`.
```

https://sembr.org/

**Review by 2e3s — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**